libcudacxx/test/libcudacxx/std/atomics/atomics.types.operations/atomics.types.operations.req/ctor.pass.cpp

```diff
@@ -60,7 +60,7 @@ struct TestFunc
       constexpr Atomic a{t};
       assert(a == t);
     }
-#if !defined(_GNUC_VER) || _GNUC_VER >= 409
```

Comment:
This change needs to be synced with #2784, because then it would change to `#if !_CCCL_COMPILER_GCC || _CCCL_GCC_VERSION >= 40900`

Reply:
Yes, I will convert this PR to draft untill #2784 is merged.

Reply:
We limited the scope of #2784, so this should be fine to go